doonut-app-external-api/src/main/kotlin/com/doonutmate/member/controller/MemberController.kt

```diff
@@ -71,4 +71,18 @@ class MemberController(
     ): Long {
         return memberProfileFacadeService.updateProfile(memberId, nameRequest.nickname, multipartFile)
     }
+
+    @Operation(summary = "프로필 수정", description = "멤버가 이름을 수정한다.")
```

Comment:
프로필명 수정으로 summary 바꿔도 좋을 것 같습니다!  

```suggestion
    @Operation(summary = "프로필명 수정", description = "멤버가 이름을 수정한다.")
```